api/lib/models/checkout-items/checkout-items-data-mapper-impl.ts

```diff
@@ -87,7 +156,7 @@ export class CheckoutItemsDataMapperImpl extends GenericDataMapper
       .left_join(this.activeHackathonDataMapper.tableName, 'h', 'c.hackathon=h.uid and h.active=1')
       .group('i.uid')
       .toParam();
-
+    query.text = query.text.concat(';');
     return from(
       this.sql.query<CheckoutItems>(query.text, [], {stream: true, cache: true }))
```

Comment:
nitpick: pass in query.values instead of [] just in case at some point the query changes